ci/patch_bazel_windows/patch-scoop

```diff
@@ -0,0 +1,51 @@
+adapted from https://github.com/lukesampson/scoop/pull/2950/files
```

Comment:
An alternative would be to simply move `bazel.exe` somewhere and put it in PATH. It’s a self-extracting standalone exe so the install process is mostly non-existent.

Reply:
Opted for hosting the binary, mostly to address the next comment, but also to avoid having to patch Scoop like that.